components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -702,27 +715,26 @@
 $xwiki.jsx.use('PhenoTips.PatientConsent')
 $xwiki.ssx.use('PhenoTips.PatientConsent')
 #if ($services.consentauthorizer.consentsGloballyEnabled())
-  (% id='consents-block' class='$xcontext.action' %)(((
+  #set($consentBaseUrl = "/rest/patients/${doc.getName()}/consents/")
+  (% id='consents-block' class='$xcontext.action' data-url='$consentBaseUrl' %)(((
     {{html clean='false' wiki='false'}}&lt;div class='patient-id hidden'&gt;${doc.getName()}&lt;/div&gt;{{/html}}
-    (% class='view-title hidden' %)(((
+    (% class='consents-title' %)(((
```

Comment:
Looks ok as is but want confirm that CSS rule for "consents-title" is omitted on purpose